.github/workflows/auto-request-review.yml

```
@@ -8,19 +8,29 @@ on:
+            echo "run=true" >> $GITHUB_OUTPUT
+          fi
 
+  auto-request-review:
+    needs: should-run
+    if: needs.should-run.outputs.run == 'true'
+    runs-on: ubuntu-latest
+    name: Auto Request Review
+    steps:
       - name: Request reviews based on configuration
-        if: ${{ always() && steps.check_owner.outcome == 'skipped' }}
+        if: ${{ !steps.check_owner.outputs.skip }}
```

actionlint (1.7.4)

property "check_owner" is not defined in object type {}

(expression)